AOJ/06/06_C.py

```diff
@@ -0,0 +1,14 @@
+tab=[[[0]*10 for i in range (0,3)]for k in range (0,4)]
+N= int(input())
+for loop in range(N):
+    house,floor,room,add=map(int,input().split())
+    tab[house-1][floor-1][room-1]+=add
+x=0
+for i in range(4):
+    if x !=0:
+        print('#'*20)
+    x+=1
```

Comment:
`i` と `x` のインデックスとしての役割が同じですので、`x` を `i` で書き直すことができます。
```suggestion
for i in range(4):
    if i !=0:
        print('#'*20)
```